packages/ui/src/Stepper/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { ReactNode } from "react";
+
+import { IStepEvent, LineStyleType, Step } from "./Step";
+
+type StepItem = {
+  activeStepIcon?: string | ReactNode;
+  label?: string;
+  step?: number | string | ReactNode;
+};
+
+interface IProperties {
+  activeIndex?: number;
+  onChange?: (event: IStepEvent) => void;
+  readOnly?: boolean;
+  lineStyle?: LineStyleType;
+  stepList?: StepItem[];
```

Comment:
I think this should be required. Stepper is meaningless without the steps.
```suggestion
  steps: StepItem[];
```